### ISSUE TITLE
Database if often lost after a macOS crash

### ISSUE THREAD

**Issue by josefglatz** [rupa/z#243]
I found already some database crash related issues here. I generelly have no torubles with it.

Except when macOS crashes, my db is gone. Does anybody else have this problem? Maybe I havae something misconfigured (I have zsh and nothing special configured so far)

**Comment by alphaCTzo7G** [comment]
+1.. This actually happens to me quite a lot.. Not sure why. It happens randomly even when I have no apparently crash.. but some changes in the configuration...

On ubuntu 16.04..

I have been thinking of putting my .z file into a git version control (with some kind of cron job based automated commit) to reduce the risk of losing the history. 

Is there a better way?

**Comment by balta2ar** [comment]
Did you try this pull request? https://github.com/rupa/z/pull/199

**Comment by alphaCTzo7G** [comment]
yeah.. I replied to your PR but haven't tried it out. Will try it out..
